spec.html

```diff
@@ -273,6 +273,40 @@ <h1>Runtime Semantics: BindingClassDeclarationEvaluation</h1>
         <p><emu-grammar>ClassDeclaration : `class` ClassTail</emu-grammar> only occurs as part of an |ExportDeclaration| and the setting of a name property and establishing its binding are handled as part of the evaluation action for that production. See <emu-xref href="#sec-exports-runtime-semantics-evaluation"></emu-xref>.</p>
       </emu-note>
     </emu-clause>
+
+  <emu-clause id="sec-coalesce-getter-setter" aoid=CoalesceGetterSetter>
+    <h1>CoalesceGetterSetter ( _element_, _other_ )</h1>
+    <emu-alg>
+      1. Assert: IsAccessorDescriptor(_other_.[[Descriptor]]) and IsAccessorDescriptor(_element_.[[Descriptor]]) are both *true*,
+      1. If _element_.[[Descriptor]] has a [[Get]] field,
+        1. Set _other_.[[Descriptor]].[[Get]] to _element_.[[Descriptor]].[[Get]].
+      1. Otherwise,
+        1. Assert: _element_.[[Descriptor]] has a [[Set]] field.
+        1. Set _other_.[[Descriptor]].[[Set]] to _element_.[[Descriptor]].[[Set]].
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-coalesce-class-elements" aoid="CoalesceClassElements">
+    <h1>CoalesceClassElements ( _elements_ )</h1>
+    <emu-alg>
+      1. Let _newElements_ be an empty List.
+      1. For _element_ in _elements_,
+        1. If _element_.[[Kind]] is `"method"` and _newElements_ contains a Record _other_ where _other_.[[Kind]] is `"method"`, SameValue(_other_.[[Key]], _element_.[[Key]]) is *true*, and _other_.[[Placement]] is _element_.[[Placement]],
+          1. If _element_.[[Decorators]] is not empty,
+            1. If _other_.[[Decorators]] is not empty, throw a *ReferenceError* exception.
+            1. Set _other_.[[Decorators]] to _element_.[[Decorators]].
+          1. If IsDataDescriptor(_element_.[[Descriptor]]) is *true* or IsDataDescriptor(_other_.[[Descriptor]]) is *true*, then
+            1. Assert: _element_.[[Key]] is not a Private Name.
+            1. Assert: _element_.[[Descriptor]].[[Configurable]] is *true*, and _other_.[[Descriptor]].[[Configurable]] is *true*.
```

Comment:
Is there no possibility that a decorator could return a nonconfigurable public data descriptor with kind “method” here?

Reply:
No, since this is only run on syntactic constructs before decorators.